docs/content/docs/try-flink/local_installation.md

```diff
@@ -46,6 +46,8 @@ to have __Java 11__ installed. To check the Java version installed, type in your
 $ java -version
 ```
 
+**Notes**: JDK 17 and above is not supported yet!!!
+
```

Comment:
In fact it is supported started from 1.18.x
Also above in this doc there is already mentioned that there should be JAva 11 downloaded
could you please give more insites why dow we need it?